source/game_sa/Clothes.cpp

```diff
@@ -67,7 +65,91 @@ int32 GetClothesModelFromName(const char* name) {
 
 // 0x5A7B30
 void CClothes::LoadClothesFile() {
-    plugin::Call<0x5A7B30>();
+    bool isRuleStarted = false;
+    auto* file = CFileMgr::OpenFile("DATA\\CLOTHES.DAT", "r");
+
+    for (auto line = CFileLoader::LoadLine(file); line; line = CFileLoader::LoadLine(file)) {
+        if (!line[0] || line[0] == '#') {
```

Comment:
```suggestion
        if (line[0] == '\0' || line[0] == '#') {
```
personal preference tho.